production/helm/loki/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: "v1"
 name: loki
-version: 0.25.0
+version: 0.26.0
```

Comment:
```suggestion
version: 0.26.1
```

Reply:
You have a point there, will update accordingly.